klein/test/test_resource.py

```diff
@@ -1014,7 +1017,19 @@ def test_urlDecodeErrorRepr(self):
             repr(_URLDecodeError(ValueError)),
         )
 
+    def test_urlDecodeErrorReprPy3(self):
+        """
+        URLDecodeError.__repr__ formats properly.
```

Comment:
"properly" is one of those words we should avoid: https://jml.io/pages/test-docstrings.html